wikitemplate.md

```diff
@@ -188,6 +189,24 @@ _Each start should take less than 7 seconds_
 - [ ] Visit https://dlptest.com/ftp-test/ and ensure that basic authentication is working by clicking on ftp://ftp.dlptest.com/ and ensuring you're prompted for credentials
 - [ ] Test that WebSockets are working by ensuring http://slither.io/ runs once "Play" has been clicked.
 
+## Tor Private Tabs
+
+- [ ] Visit https://check.torproject.org in a Tor private tab, ensure its shows success message for using a Tor exit node 
+- [ ] Open a Tor private tab and toggle Tor switch , visit https://check.torproject.org, ensure it shows failure message for not using Tor exit node
+- [ ] Open a normal tab and serach `Check Tor`, open the https://check.torproject.org link in a Tor private tab, ensure tab uses Tor exit node
+- [ ] Visit https://check.torproject.org in a Tor private tab, note down exit node IP address, Click `New circuit for this site` in shields, ensure the exit node IP address changes after page is reloaded 
+- [ ] Visit https://protonirockerxow.onion/ in a Tor private tab, ensure login page is shown
+- [ ] Open Tor private tab and toggle Tor switch, visit https://protonirockerxow.onion/ ensure website doesn't load
+- [ ] Visit https://browserleaks.com/geo, ensure location isn't shown
```

Comment:
should add some more context here, should probably say `Visit https://browserleaks.com/geo in a Tor private tab`

Reply:
fixed